scripts/licenses/licenses/library.go

```diff
@@ -0,0 +1,175 @@
+// Copyright 2019 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package licenses
+
+import (
+	"fmt"
+	"go/build"
+	"regexp"
+	"sort"
+	"sync"
+
+	"github.com/golang/glog"
+)
+
+var (
+	licenseRegexp = regexp.MustCompile(`^LICENSE(\.(txt|md))?$`)
```

Comment:
move `licenseRegexp` to `find.go` (as it's only used there)?

Reply:
Done.